dataclients/kubernetes/definitions/testdata/errorwrapdata/errors.log

```diff
@@ -4,3 +4,7 @@ single predicate expected at "Path(\"/foo\") && Method(\"GET\")"
 single predicate expected at ""
 single filter expected at "inlineContent(\"/foo\") -> status(200)"
 single filter expected at " "
+backend address "http://example.com/foo" contains path
+backend address "http://example.com/foo/bar" contains path
+backend address "http://example.com/foo/" contains path
```

Comment:
Should we also add the following failures?
```
http://example.com
http://example.com/
https://example.com/foo
example.com/
example.com/foo
```

Reply:
why `http://example.com` is a failure?

I think `https://example.com/foo` is already covered by `http://example.com/foo`

Will add some of them later, I still didn't make tests work.

Reply:
> why http://example.com is a failure?

I just was thinking that explicit https configuration is not allowed as well.

Reply:
All we care about in this PR if the address has path or not. Maybe we need to add validation later for https but I don't see why it's not allowed, maybe we talk to east-west route without certificate.

Reply:
What should work is: `http://user:pass@example.com`